web/ui/dashboard/src/app/private/components/create-source/create-source.component.html

```diff
@@ -205,6 +205,14 @@ <h2 class="font-semibold mb-20px">Configure SQS</h2>
 	</div>
 </form>
 
+<convoy-token-modal
+	title="Source URL Generated"
+	description="Use this URL in your source platform to receive your webhook events."
+	notificationText="Source URL has been copied to your clipboard."
+	[token]="sourceData.url"
+	(closeModal)="showSourceUrl = false; onAction.emit({ action: this.action, data: sourceData })"
+	*ngIf="showSourceUrl"
+></convoy-token-modal>
```

Comment:
Since this PR is a UI update, we should fix this modal thing once and for all. We should only have modal components as containers and only have something like this only when a specific usecase is reused more than twice. @Oluwadaminiola

Reply:
We're using token-modal for create source?